server/jetstream_consumer_test.go

```diff
@@ -1402,6 +1403,120 @@ func TestJetStreamConsumerPedanticMode(t *testing.T) {
 		}
 	}
 }
+
+func TestJetStreamConsumerStuckAckPending(t *testing.T) {
+	s := RunBasicJetStreamServer(t)
+	defer s.Shutdown()
+
+	nc, js := jsClientConnect(t, s)
+	defer nc.Close()
+
+	type ActiveWorkItem struct {
+		ID     string
```

Comment:
Looks like all usage of `ID` is via `strconv.Itoa` etc, why not just make it an `int` and then use the `%d` formatter instead of `%v`?

Reply:
Sure!